kobo/apps/organizations/models.py

```diff
@@ -84,6 +84,20 @@ def owner_user_object(self) -> 'User':
         except ObjectDoesNotExist:
             return
 
+    @property
+    @cache_for_request
+    def is_mmo(self):
```

Comment:
Please sort alphabetically. Ensure to pull the base branch, I may have changed few lines of code already.